src/actions/classifier.js

```diff
@@ -25,6 +25,11 @@ export function startNewClassification(workflowID) {
     }).then(() => {
       return dispatch(setNeedsTutorial())
     }).then(() => {
+      if (getState().classifier.guide[workflowID] !== undefined) {
+        return
+      }
+      return dispatch(fetchFieldGuide(workflowID))
+    }).then(() => {
```

Comment:
Is it possible to combine all of these `.then()` statements rather than having a separate call and return for each? Might save a few lines of code and clean up this component a bit.